frontend/src/assets/fonts/README.md

```diff
@@ -1,19 +1,25 @@
 # Export PDF Language Support
 
-To support multiple languages with special characters we need to register fonts and set the fontFamily (example in ExportContent.tsx)
-Download font families tts or otf files from https://fonts.google.com/noto to assets/fonts/
+To support multiple languages with special characters we need to register fonts and set the font-family -
+see [ExportContent.tsx](../../components/ExportChat/ExportContent.tsx). You can download additional font families
+(ttf or otf) from https://fonts.google.com/noto and place in src/assets/fonts.
 
-Currently can only use a single file at a time, so we can merge multiple using script from https://github.com/notofonts/nototools/blob/main/nototools/merge_fonts.py.
+Currently only a single font-family can be used at a time, so multiple fonts can be merged using this script:
+https://github.com/notofonts/nototools/blob/main/nototools/merge_fonts.py.
 
 The current CombinedFont.ttf contains:
 
 - NotoSans-Regular.ttf
-- NotoSerifDevanagari-Regular.ttf
 - NotoKufiArabic-Regular.ttf
+- NotoSansHebrew-Regular.ttf
 - NotoSansThai-Regular.ttf
 - NotoSerifBengali_Condensed-Regular.ttf
+- NotoSerifDevanagari-Regular.ttf
 - NotoSerifGurmukhi-Regular.ttf
-- NotoSansHebrew-Regular.ttf
+
+Japanese and Chinese font packs are not included in the CombinedFonts family as they are prohibitively large, so if you are
+running the application locally and wish to export chats in either of those fonts, you will need to modify the code in
+[ExportContent.tsx](../../components/ExportChat/ExportContent.tsx) accordingly.
```

Comment:
worth mentioning explicitly that these will need to be downloaded? A user might assume they're in the files somewhere given that there's commented out code that mentions them.

Reply:
Only if we decide to remove those two fonts from the repo 😉

Reply:
Yep, done.